lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -43,22 +43,19 @@ const INITIAL_DIALOG_STATE = {
 };
 
 const CanvasNoFiles = (
-  <Fragment>
-    <div className="FileList__no-files">
-      There are no available PDF files in the current course.
-    </div>
+  <div className="FileList__no-files">
```

Comment:
Technically, using BEM syntax, I _believe_ this classname may be more correct as `FileList--no-files`, if I'm understanding its meaning correctly: this is a `FileList` in the context of `no-files` (i.e. would use "Modifier" syntax).

Reply:
Using a component `__` is correct as this is a sub-element of the `FilesList`. Maybe `FileList__no-files-message` might be better as it is a more noun-y name.